package/MDAnalysis/coordinates/base.py

```diff
@@ -2069,6 +2078,54 @@ def _apply_transformations(self, ts):
         return ts
 
 
+class _AsciiPickle(object):
```

Comment:
Docstring explaining what these three classes do / when they should be useful would be very helpful here (especially when it comes to the implementation of future readers).

Reply:
Document everything, even if you make it a private class. You're working on the core code, for years to come, developers will have to work with what you're writing here, so be super-clear about 
- what your intent is (how it _should_ behave)
- how it interacts with other parts of the code
- what the caveats, _requirements_, and  _implicit assumptions_ are
- any TODOS/known issues